docs/tempo/website/metrics-generator/app-performance-mgmt.md

```diff
@@ -0,0 +1,167 @@
+---
+aliases:
+- /docs/tempo/latest/server_side_metrics/service_graphs/
+title: Application performance management
+menuTitle: APM dashboard
+weight: 200
+---
+
+# Application performance management dashboard
+
+<span style="background-color:#f3f973;">This is an experimental feature. For more information about how to enable it, continue reading.</span>
+
+Application performance management (APM) uses data gathered from services, agents, systems, and microservices to visualize and identify areas of concern.
+
+The Grafana APM dashboard visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+Once the requirements are set up, this pre-configured dashboard is immediately available.  
+
+You can use the APM dashboard to:
+
+* Discover spans which are consistently erroring and the rates at which they occur
+* Get an overview of the overall rate of span calls throughout your services
+* Determine how long the top 10% slowest queries in your service take to complete
+* Examine all traces that contain spans of particular interest based on rate, error and duration values (RED signals)
+
+<p align="center"><img src="../../getting-started/apm-overview.png" alt="APM dashboard"></p>
+
+## Requirements to enable the APM dashboard
+
+You have to enable metric graph and service graph generation on the Grafana backend so metrics are generated as traces are ingested.
+
+To use the APM dashboard, you need:
+
+* Tempo or Grafana Cloud Traces with the metrics generator enabled and configured to send data to a Prometheus compatible metrics
+* [Services graphs]({{< relref "../metrics-generator/service_graphs/#how-to-run" >}}), which are enabled by default in Grafana
+* [APM table enabled](https://grafana.com/docs/grafana/next/datasources/tempo/) within the service graph tab (use the `tempoApmTable` feature flag in Grafana to enable)
+* [Span metrics]({{< relref "../metrics-generator/span_metrics/#how-to-run" >}}) enabled in your Tempo data source configuration
+
+The APM dashboard can be generated with the metrics-generator or Grafana Agent.
```

Comment:
If this is the case, doesn't this relax requirement (1)?

Namely 
```
Tempo or Grafana Cloud Traces with the metrics generator enabled and configured to send data to a Prometheus compatible metrics
```

Maybe requirement becomes "tempo or GCT with metrics generator...or grafana agent (with whatever transforms/extra stuff is required on the agent)"?

Reply:
This is one of the questions I have about this document. On Thursday, I learned that the APM dashboard will work with metrics-generator and/or Grafana Agent. I wrote this thinking just metrics-generator. It needs to be reviewed for that difference.

Reply:
I've made a suggestion above by adding the metrics-generator and Agent in the first requirements bullet.

Reply:
I believe that the service graph and APM table and service graph will work with grafana agent metrics or tempo metrics, but it really should be tested.

Reply:
@joe-elliott APM table is marked as an experimental feature in this doc. Do you want the statement (first bullet) to stay as is or should it be reworded to only point to configuration that we have tested? And is that config metrics generator primarily?

Reply:
Clarified with Joe in Slack. Statement should be okay.